frontend/src/app/components/transaction/transaction.component.ts

```diff
@@ -495,6 +517,7 @@ export class TransactionComponent implements OnInit, AfterViewInit, OnDestroy {
 
   @HostListener('window:resize', ['$event'])
   setGraphSize(): void {
+    this.isMobile = window.innerWidth < 850;
```

Comment:
Usually better to use media query for showing/hiding html elements

Reply:
We need to add/remove the actual table row markup element to get the correct alternating table stripes.

`display: none` via CSS media query isn't enough.